### ISSUE TITLE
sync failed after update to 2.6.9

### ISSUE THREAD

**Issue by oldflag2333333** [laurent22/joplin#5870]
## Environment

Joplin version: 2.6.9 portable
Platform: windows10

## Steps to reproduce

After update to 2.6.9,  sync does not work,and i dont know the version of joplin client before update, but im sure it works properly.Im using my own joplin server, here is my joplin server config.
```
# This is a sample docker-compose file that can be used to run Joplin Server
# along with a PostgreSQL server.
#
# All environment variables are optional. If you don't set them, you will get a
# warning from docker-compose, however the app should use working defaults.

version: '3'

services:
    db:
        image: postgres:13.1
        volumes:
            - ./data/postgres:/var/lib/postgresql/data
        ports:
            - "5432:5432"
        restart: unless-stopped
        environment:
            - POSTGRES_PASSWORD=password
            - POSTGRES_USER=joplin
            - POSTGRES_DB=joplin
    app:
        image: joplin/server:latest
        depends_on:
            - db
        ports:
            - "22300:22300"
        restart: unless-stopped
        environment:
            - APP_PORT=22300
            - APP_BASE_URL=https://mydomain
            - DB_CLIENT=pg
            - POSTGRES_PASSWORD=password
            - POSTGRES_DATABASE=joplin
            - POSTGRES_USER=joplin
            - POSTGRES_PORT=5432
            - POSTGRES_HOST=db
```

## Describe what you expected to happen
How to solve this issue?

## Logfile
```
2021-12-17 23:19:35: Synchronizer: "Error: Error 412 Precondition Failed: <!doctype html>
<html>
<head>
	<title>Joplin Server - Error</title>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1" />
	<link rel="stylesheet" href="https://mydomain/css/bulma.min.css" crossorigin="anonymous">
		<!-- <link rel="stylesheet" href="https://mydomain/css/bulma-prefers-dark.min.css" crossorigin="anonymous"> -->
	<link rel="stylesheet" href="https://mydomain/css/main.css" crossorigin="anonymous">
	<link rel="stylesheet" href="https://mydomain/css/fontawesome/css/all.min.css" crossorigin="anonymous">
	<script src="https://mydomain/js/jquery.min.js"></script>
	<script src="https://mydomain/js/main.js"></script>
</head>
	<body class="page-error">
		<main class="main">
			<div class="container">

				<script>
					onDocumentReady(function() {
						const buttons = document.getElementsByClassName('close-notification-button');
						
						for (const button of buttons) {
							button.addEventListener(
Code: 412
Error: Error 412 Precondition Failed: <!doctype html>
<html>
<head>
	<title>Joplin Server - Error</title>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1" />
	<link rel="stylesheet" href="https://mydomain/css/bulma.min.css" crossorigin="anonymous">
		<!-- <link rel="stylesheet" href="https://mydomain/css/bulma-prefers-dark.min.css" crossorigin="anonymous"> -->
	<link rel="stylesheet" href="https://mydomain/css/main.css" crossorigin="anonymous">
	<link rel="stylesheet" href="https://mydomain/css/fontawesome/css/all.min.css" crossorigin="anonymous">
	<script src="https://mydomain/js/jquery.min.js"></script>
	<script src="https://mydomain/js/main.js"></script>
</head>
	<body class="page-error">
		<main class="main">
			<div class="container">

				<script>
					onDocumentReady(function() {
						const buttons = document.getElementsByClassName('close-notification-button');
						
						for (const button of buttons) {
							button.addEventListener(
    at newError (C:\Users\heiyi\AppData\Local\Temp\22MXQNwz0935WVLl6HzdD9BvIA2\resources\app.asar\node_modules\@joplin\lib\JoplinServerApi.js:192:28)
    at JoplinServerApi.<anonymous> (C:\Users\heiyi\AppData\Local\Temp\22MXQNwz0935WVLl6HzdD9BvIA2\resources\app.asar\node_modules\@joplin\lib\JoplinServerApi.js:232:27)
    at Generator.throw (<anonymous>)
    at rejected (C:\Users\heiyi\AppData\Local\Temp\22MXQNwz0935WVLl6HzdD9BvIA2\resources\app.asar\node_modules\@joplin\lib\JoplinServerApi.js:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)"
```

**Comment by laurent22** [comment]
You need to upgrade Joplin Server to 2.6

**Comment by Derkades** [comment]
Any plans to mark the latest Joplin Server 2.6 as stable? Currently it's only available under `2.6-beta` on docker hub.

**Comment by goaround** [comment]
@laurent22 The problem is 2.6.10 fails on npm update https://github.com/laurent22/joplin/issues/5896

An updated docker image would be really nice. Especially because on iOS a downgrade of the app is not possible, and I can't access my attachments there anymore.
